build.sh

```diff
@@ -198,11 +202,21 @@ if buildAll || hasArg libcudf; then
 
     # Record build times
     if [[ -f "${LIB_BUILD_DIR}/.ninja_log" ]]; then
-        echo "Formatting build times"
+        echo "Formatting build metrics"
         python ${REPODIR}/cpp/scripts/sort_ninja_log.py ${LIB_BUILD_DIR}/.ninja_log --fmt xml > ${LIB_BUILD_DIR}/ninja_log.xml
-        message="$FILES_IN_CCACHE <p>$PARALLEL_LEVEL parallel build time is $compile_total seconds"
-        echo "$message"
-        python ${REPODIR}/cpp/scripts/sort_ninja_log.py ${LIB_BUILD_DIR}/.ninja_log --fmt html --msg "$message" > ${LIB_BUILD_DIR}/ninja_log.html
+        MSG="<p>"
```

Comment:
I missed this the first time, but I am going to push back on this logic being implicitly on inside `build.sh` and not something developers/ci opts-in to.

The logic around this presumes multiple things like:

- ccache existance means it was used ( not true for our sccache users )
- static build support, since this logic implicitly presumes `libcudf.so` exists
- users are okay with global ccache stats being cleared

By providing a flag to opt-in to this behavior, we can both support edge case users and make it clearer to all what happens when you run `./build.sh`

Reply:
Thanks Robert. I think this makes sense. 
The opt-in mechanism you mentioned would be a parameter(s) passed to `/build.sh` ?
I can think of two parameters at the moment: (1) generate build report and (2) include ccache statistics
I'll change the script to only include the size of `libcudf.so` if it exists.

Reply:
Yes, I was imaging two flags, one for build times, and the other for build cache stats. Maybe the names `--build-times` and `--build-cache-stats`?